src/array/ops/count.rs

```diff
@@ -0,0 +1,32 @@
+use std::sync::Arc;
+
+use arrow2;
+
+use crate::{array::DataArray, datatypes::*, error::DaftResult};
+
+use super::DaftCountAggable;
+
+impl<T> DaftCountAggable for &DataArray<T>
+where
+    T: DaftDataType,
+{
+    type Output = DaftResult<DataArray<UInt64Type>>;
+
+    fn count(&self) -> Self::Output {
+        let arrow_array = &self.data;
+        let result_arrow_array = match arrow_array.len() {
+            0 => arrow2::array::PrimitiveArray::from([]),
```

Comment:
I believe this is different compared to pandas and pyarrow which return 0 for an empty array, which i think also makes more sense

Reply:
aww good catch! I think we changed the behaviors of the nulls in aggregations then. @jaychia Why did we deviate from what pyarrow and pandas does?

Reply:
We decided on this after drawing up that matrix of behavior across empty/nulls/all-nulls. I don't think we checked that against pyarrow/pandas/postgres behavior actually.

Seems like PostgreSQL does return one empty row for global aggs vs no rows for a group-by agg on an empty table though. We can just follow the same behavior then and change our tests